cmake/Modules/ExternalJSONC.cmake

```diff
@@ -48,13 +48,10 @@ if (EXISTS ${PROJECT_SOURCE_DIR}/lib/jsonc/json.h)
      )
 
     set(${LIB_NAME}_INTERNAL TRUE)
-    set(${LIB_NAME}_INTERNAL_INCLUDE_DIR "${PROJECT_BINARY_DIR}/include/json-c/" INTERNAL)
-    set(${LIB_NAME}_INTERNAL_LIBRARY "${PROJECT_BINARY_DIR}/lib/libjson-c.so" INTERNAL)
+    set(${LIB_NAME}_INTERNAL_INCLUDE_DIR "${PROJECT_BINARY_DIR}/include/json-c/")
```

Comment:
Is this related to the unit tests?
If not, can you extract the changes of this file to a separate commit?

Reply:
Fixed